lib/SILOptimizer/Mandatory/TFCanonicalizeCFG.cpp

```diff
@@ -152,6 +160,13 @@ namespace {
     /// processLoops fills in this mapping: it is keyed by the preheader block
     /// of each loop, and points to the region produced for it.
     llvm::DenseMap<SILBasicBlock*, WhileLoopSESERegion*> loopPreheaders;
+
+    /// Map that keeps track of the underlying SESERegionTree for the
+    /// FunctionSESERegion that starts at the given block.
```

Comment:
pls extend the comment on what is the map key, and what is the second component in the pair of the value component.